scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementSample.java

```diff
@@ -0,0 +1,202 @@
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.surface_placement;
+
+import java.util.Arrays;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.LayerSceneProperties.SurfacePlacement;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+import com.esri.arcgisruntime.symbology.TextSymbol;
+import com.esri.arcgisruntime.symbology.TextSymbol.HorizontalAlignment;
+import com.esri.arcgisruntime.symbology.TextSymbol.VerticalAlignment;
+
+public class SurfacePlacementSample extends Application {
+
+  private SceneView sceneView;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+
+      // create stack pane and JavaFX app scene
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+      fxScene.getStylesheets().add(getClass().getResource("/style.css").toExternalForm());
+
+      // set title, size, and add JavaFX scene to stage
+      stage.setTitle("Surface Placement Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a scene and add a basemap to it
+      ArcGISScene scene = new ArcGISScene(Basemap.Type.IMAGERY);
+
+      // add the SceneView to the stack pane
+      sceneView = new SceneView();
+      sceneView.setArcGISScene(scene);
+
+      // add a camera and initial camera position
+      sceneView.setViewpointCamera(new Camera(53.05, -4.01, 1115, 299, 88, 0));
+
+      // add base surface for elevation data
+      Surface surface = new Surface();
+      ArcGISTiledElevationSource elevationSource = new ArcGISTiledElevationSource(
+          "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer");
+      surface.getElevationSources().add(elevationSource);
+      scene.setBaseSurface(surface);
+
+      // create overlays with elevation modes
```

Comment:
I think since we're making the effort to rename this "Surface placement" rather than "Elevation Mode" to be more in keeping with the API naming, this comment could be replaced with 
```suggestion
      // create overlays with surface placement types
```